elements/Toggle/Toggle.stories.js

```diff
@@ -0,0 +1,44 @@
+import lng from '@lightningjs/core';
+
+import Toggle from '.';
+import mdx from './Toggle.mdx';
+
+export default {
+  title: 'Elements / Toggle',
+  parameters: {
+    docs: {
+      page: mdx
+    }
+  }
+};
+
+export const Basic = args =>
+  class Basic extends lng.Component {
+    static _template() {
+      return {
+        flex: { direction: 'column' },
+        text: { fontSize: 24, text: 'Press Enter' },
+        Toggle: {
+          y: 40,
+          type: Toggle,
+          checked: args.checked,
+          onEnter: toggle => {
+            alert('Sample onEnter override');
```

Comment:
Could this be logged as an action instead of alert?

Reply:
yes it could!